Procfile

```diff
@@ -1,2 +1,3 @@
+release: rake db:migrate
```

Comment:
I think we should use the bin stub to be consistent with the other processes. Also `bin/rails` is the [new default for running commands](http://guides.rubyonrails.org/command_line.html#bin-rails).

```
release: bin/rails db:migrate
```

Reply:
I was being consistent with app.json, but OK. I wonder how long this 'default' will take to become the default ;-)

Reply:
I'm fighting the good fight ;-)